### PR TITLE
Fix `Lint/HandleExceptions` and `Lint/RescueException` offenses reported by rubocop.

### PR DESCRIPTION
**Pull request by tejasbubane**

### CODE REVIEW

File: lib/api/routes/exercises.rb [hunk 1]
```diff
@@ -12,7 +13,7 @@ class Exercises < Core
           result = Homework.new(current_user).all.to_json
           LifecycleEvent.track('fetched', current_user.id)
           result
-        rescue Exception => e
+        rescue StandardError => e
```
Comment (kytrinyx) [inline lib/api/routes/exercises.rb]: I explicitly want to handle exception here, not standard error, in order to send it to bugsnag and provide a semi-friendly message. There's some syntax to explicitly allow certain violations, right?

File: lib/exercism/language.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 class Language
   def self.of(key)
     ExercismWeb::Presenters::Tracks.find(key.to_s).language
-  rescue Exception => e
```
Comment (kytrinyx) [inline lib/exercism/language.rb]: Same here--I want bugsnag to know about all exceptions.

**Comment by tejasbubane** [comment]
We can disable `Lint/RescueException` in this file only by adding:

```
# rubocop:disable Lint/HandleExceptions, Lint/RescueException
```

Or should we can allow it throughout the project by adding it in `.rubocop.yml`?

**Comment by kytrinyx** [comment]
I don't want to disable it everywhere, just in that file.

**Comment by tejasbubane** [comment]
@kytrinyx Done. Disabled the cop for those two files.

**Comment by kytrinyx** [comment]
Nice, thanks.

**Pull request merged**